services/jsdelivr/jsdelivr-base.js

```diff
@@ -18,19 +18,19 @@ const periodMap = {
 class BaseJsDelivrService extends BaseJsonService {
   static render({ period, hits }) {
     return {
-      message: `${metric(hits)} hits/${periodMap[period]}`,
+      message: `${metric(hits)}/${periodMap[period]}`,
     }
   }
 
   static get defaultBadgeData() {
     return {
       label: 'jsdelivr',
-      color: 'orange',
+      color: 'brightgreen',
```

Comment:
Thoughts on using `informational` here if the color will always be the same (I don't think the services are using the dynamic `downloadCount` from `color-formatters.js`?

Reply:
Yea, probably they should use `downloadCount`, though I think we ought to move `renderDownloadBadge` out of `nuget-helpers` and use it more widely. I didn't really want to tackle that here…

I don't feel strongly, though it seems short of using `downloadCount`, making these brightgreen is a step toward more consistentcy with the other download badges.

Reply:
Eh, it's not that hard to call `downloadCount`. I'll update.